cirq/circuits/circuit.py

```diff
@@ -95,7 +95,7 @@ class Circuit:
 
     @deprecated_parameter(
         deadline='v0.8',
-        fix='Pass contents positionally or using the "contents=" keyword.',
+        fix='Pass circuit contents positionally (without a keyword).',
```

Comment:
explain?

Reply:
This was an oversight on my part when writing the original message. It turns out you can't pass `*args` arguments by name.

Reply:
oh duh. For some reason my brain kept reading it as `**contents` in the signature